layouts/index.html

```diff
@@ -1,591 +1,262 @@
+              class="flex h-full flex-col gap-2 rounded border border-gray-light-100 bg-white p-4 drop-shadow-sm dark:border-gray-dark-400 dark:bg-gray-dark-200">
               <div class="flex items-center gap-4">
+                <span class="icon text-gray-light dark:text-gray-dark" data-icon="{{ .icon }}"></span>
                 <div>
-                  <div class="text-xl leading-snug text-gray-light-800 dark:text-white">
-                    Meet the captains
-                  </div>
+                  <div class="text-xl leading-snug">{{ .title }}</div>
                 </div>
               </div>
-              <div class="leading-snug text-gray-light-500 dark:text-gray-dark-700">
-                Select members of the community that are both experts in their field and are passionate about sharing
-                their Docker knowledge with others.
+              <div class="flex h-full flex-col justify-between gap-4">
+                {{ .description | markdownify }}
               </div>
             </div>
-          </a>
+          </div>
+          {{ end }}
         </div>
-        <div>
-          <a class="h-full" href="/get-started/resources/">
-            <div
-              class="flex h-full flex-col gap-2 rounded border border-gray-light-100 bg-white p-4 drop-shadow-sm hover:border-gray-light-200 hover:drop-shadow-lg dark:border-gray-dark-400 dark:bg-gray-dark-200 hover:dark:border-gray-dark">
-              <div class="flex items-center gap-4">
-                <div>
-                  <div class="text-xl leading-snug text-gray-light-800 dark:text-white">
-                    Educational resources
-                  </div>
-                </div>
-              </div>
-              <div class="leading-snug text-gray-light-500 dark:text-gray-dark-700">
-                Expand your understanding of Docker with these additional resources.
-              </div>
-            </div>
-          </a>
+      </div>
+    </div>
+
+    <div
+      class="relative flex h-[334px] items-center justify-center overflow-hidden bg-gray-light-100 dark:bg-gray-dark-200 sm:h-full sm:py-8">
+      <div class="w-[1200px] px-4">
+        <div class="flex max-w-[66%] flex-col gap-8 sm:max-w-full">
+          <h2 class="text-2xl">Community resources</h2>
+          <p>
+            Find fellow Docker enthusiasts, engage in insightful discussions,
+            share knowledge, and collaborate on projects. Our official and
+            unofficial communities offer a rich online experience for
+            developers to create valuable connections that challenge and
+            inspire!
+          </p>
+          <div class="flex gap-4 sm:flex-col">
+            <a href="https://forums.docker.com/"
+              class="cursor-pointer rounded bg-white px-4 py-2 text-blue-light hover:bg-gray-light-100 dark:bg-gray-dark-400 dark:text-white dark:hover:bg-gray-dark-500">
+              Forum
```

Comment:
The CTA "Join Docker Slack" sounds good. I think we should update the other two CTAs for consistency. Maybe "Visit Docker Forum" and "Find your Docker Captain"?